src/bin/cargo/commands/config.rs

```diff
@@ -5,7 +5,7 @@ pub fn cli() -> App {
     subcommand("config")
         .about("Inspect configuration values")
         .after_help("Run `cargo help config` for more detailed information.\n")
-        .setting(clap::AppSettings::SubcommandRequiredElseHelp)
+        .subcommand_required(true)
```

Comment:
I'd recommend also calling `arg_required_else_help(true)`.  Its like `SubcommandRequiredElseHelp` but only shows help when `argc == 1` so you still get quick access to help but if you leave pass in args but leave off a subcommand, you get an error specifically about that.